cmdLine/src/main/scala/topgun/cmdline/FileParser.scala

```diff
@@ -17,20 +13,19 @@ class FileParser(file: File, cmdLine: JfrParseCommandLine, totals: Totals, confi
 
   def parse(): Unit = {
 
+
+
     println(s"*** File $file")
     try {
-      val recording = FlightRecordingLoader.loadFile(file)
-      for (t <- recording.getEventTypes.asScala) {
-        println(t.getName)
-      }
-      val view = recording.createView
-      view.asScala.find(_.getEventType.getName == "GC TLAB Configuration") match {
+      val recording = new RecordingFile(file.toPath);
+
+      val view = RecordingFile.readAllEvents(file.toPath);
+      view.asScala.find(_.getEventType.getLabel == "GC TLAB Configuration") match {
         case Some(e) => handleTlabConfiguration(e)
-        case _ => ???
+        case _ => {  }
```

Comment:
If we dont have a TLAB configuration then we should fail. 
Some better error message though - file XX didnt have event  "" etc 

we also need to ensure that all files have the same settings. 

We need this setting because we need to know how big a TLAB is

Reply:
The TLAB allocation event now has a size attribute, so it looks like TLABs are variable size

If so then we don't need this, but the size of each TLAB needs to be used, rather then the configured **minimum** value